docs/metrics/prometheus/README.md

```diff
@@ -138,6 +137,23 @@ The list of metrics and its definition are as follows. (NOTE: instance here is o
 - `minio_version_info`: Current MinIO version with commit-id.
 - `s3_ttfb_seconds`: Histogram that holds the latency information of the requests.
 
+Apart from above metrics, MinIO also exposes below mode specific metrics
+
+### Cache specific metrics
+
+MinIO Gateway instances enabled with Disk-Caching expose caching related metrics.
+
+- `cache_data_served`: Total number of bytes served from cache.
+- `cache_hits_total`: Total number of cache hits.
+- `cache_misses_total`: Total number of cache misses.
+
+### S3 Gateway specific metrics
+
+MinIO S3 Gateway instance exposes metrics related to Gateway communication with AWS S3.
+
+- `gateway_s3_requests`: Total number of requests made to AWS S3.
```

Comment:
Can we also explain about the `Method` label that we added now?

Reply:
Updated docs.